trappy/plotter/Constraint.py

```diff
@@ -276,6 +276,13 @@ def _expand(self):
                    Len[columns] != 1
                    Len[templates] != 1
             )
+
+            TODO  is the following correct?
+            Permute(
+                   Len[traces] != 1
+                   Len[columns] = 1
```

Comment:
Yes, this is correct and should work when the same column is present in all the events (templates). I agree templates was a really bad name for the variable :(